services/client/src/api/pools/useFetchHypePools.tsx

```diff
@@ -1,48 +1,73 @@
-import axios from 'axios';
-import { useInfiniteQuery } from 'react-query';
-import { API, FetchHypesFilter, PoolPaginate } from '../types';
+import { FetchHypesFilter } from '../types';
+import { useQuery } from 'urql';
 
-const hypePoolsPerPage = 6;
+const hypePoolsPerPage = 3;
 
-const computePage = (page: number, search?: string): FetchHypesFilter => {
-  const take = hypePoolsPerPage;
+const computeFilters = (page: number, search?: string): FetchHypesFilter => {
+  const first = hypePoolsPerPage;
   const skip = (page - 1) * hypePoolsPerPage;
-  return {
-    take,
+
+  const filters: FetchHypesFilter = {
+    first,
     skip,
-    search,
   };
-};
 
-const fetchPools = async (params: FetchHypesFilter) => {
-  const url = `${API}/pools`;
-  try {
-    const { data } = await axios.get(url, { params });
-    return data as PoolPaginate;
-  } catch (err) {
-    console.log('Error in fetchPools: ', err);
+  if (search) {
+    filters.text = search;
   }
+
+  return filters;
 };
 
-export const useFetchHypePools = (search?: string) => {
-  const { data, isLoading, isFetchingNextPage, hasNextPage, fetchNextPage } = useInfiniteQuery(
-    ['pools', search],
-    ({ pageParam = 1 }) => fetchPools(computePage(pageParam, search)),
-    {
-      getNextPageParam: (lastPage, allPages) => {
-        const maxPages = lastPage?.total / 5;
-        const nextPage = allPages.length + 1;
-        return nextPage <= maxPages ? nextPage : undefined;
-      },
-      select: (data) => ({ ...data, pages: data.pages.flatMap((page) => page?.data) }),
-    },
-  );
+const poolsSearchQuery = `
```

Comment:
I would much rather like to have these as attributes to an object like 
```suggestion
const HYPEPOOL.searchQuery = `
```

Also, exporting it to a query collector would make our lives easier I believe.